src/main/java/org/myrobotlab/service/IntegratedMovement.java

```diff
@@ -551,11 +551,11 @@ public void setDHLink(String arm, String name, double d, double theta, double r,
     }
   }
 
-  public void setDHLink(String arm, ServoControl servo, double d, double theta, double r, double alpha) {
+  public void setDHLink(String arm, Servo servo, double d, double theta, double r, double alpha) {

```

Comment:
why the change from ServoControl to Servo here?

Reply:
it should be ServoControl - but I was struggling elsewhere, as Servo implemented some critical  methods that were not in the interface ... I'll correct it .

Reply:
Ah .. another reason is larger reason is all the "IK" awareness Servo has .. I "**don't**" want that propagated to the interface ... so this IK class has to use Servo directly